exporters/memory/include/opentelemetry/exporters/memory/in_memory_span_exporter_factory.h

```diff
@@ -3,7 +3,7 @@
 
 #pragma once
 
-#include "opentelemetry/exporters/memory/in_memory_span_data.h"
+#include "opentelemetry/exporters/memory/in_memory_span_exporter.h"
```

Comment:
Why was this include changed ?

The whole point of the factory class is to hide the implementation,
so there should be no dependencies on the in_memory_span_exporter.h header.

Reply:
Good catch. I think `InMemorySpanData` class definition should be moved to `in_memory_span_data.h`.

Reply:
Thanks! Comments resolved.